.maintain/reset-spec/app.py

```diff
@@ -108,44 +122,46 @@ def make_native(docker_img, out_file, chain, runtime):
     tmp_dir = os.path.join("/tmp/", str(uuid.uuid1()))
     os.mkdir(tmp_dir)
 
-    if args.westend:
```

Comment:
We still have the `--westend` option but we don't check anymore for it?

Reply:
good catch! Fixed in 31fa889